indico/modules/events/layout/forms.py

```diff
@@ -161,6 +161,8 @@ class MenuUserEntryFormBase(IndicoForm):
     title = StringField(_("Title"), [DataRequired()])
     is_enabled = BooleanField(_("Show"), widget=SwitchWidget())
     new_tab = BooleanField(_("Open in a new tab"), widget=SwitchWidget())
+    registered_only = BooleanField(_("Restricted"), description=_("Visible to registered users only."),
```

Comment:
we usually put the `description` kwarg last in wtforms

Reply:
I would only show this option for menu items corresponding to pages. Because for most other menu items, their visibility do not imply access restrictions, so being able to hide e.g. "Timetable" for unregistered users would be misleading, since you can still access that by URL.

Reply:
Isn't it only showing for the custom items that way, just like the "open in a new tab" option?

Reply:
You're right, it's only used for pages and links. And for external links, it's actually nice to be able to restrict it as well (if someone wants to put e.g. a zoom url there).